### DISCUSSION TITLE
请问这个编辑器字体怎么开启?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45713] [Q&A]
https://ant-design.antgroup.com/components/upload-cn

<img width="894" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/a133fc28-cf2d-4b5a-8e5b-90e208d75dbe">

**Comment by Wxh16144** [comment]
https://github.com/ant-design/ant-design/issues/45625

**Reply by vaynevayne** [reply]
让我看什么?

**Reply by Wxh16144** [reply]
这是应该是一个 bug， 看我发的 issue 地址，顺便看看解法，应该是一个 box-shadow 实现的

**Reply by vaynevayne** [reply]
好吧, 我还以为是字体呢, 还想在 vscode 用用
